src/config/process.ts

```diff
@@ -101,3 +101,8 @@ export const LND_HEALTH_REFRESH_TIME_MS = parseInt(
   process.env.LND_HEALTH_REFRESH_TIME_MS || "20000",
   10,
 )
+
+export const LND1_LOOP_TLS = process.env.LND1_LOOP_TLS ?? ""
```

Comment:
you should error if this is not present but necessary

something like:
```
const getLnd1LoopTls = () => {
   if (!process.env.LND1_LOOP_TLS) throw new Config Error
    return process.env.LND1_LOOP_TLS
}
```

Reply:
fixed in https://github.com/GaloyMoney/galoy/pull/1396/commits/146cb8791c7fc8ac731aad10585fb98eb3486ab1